udata/tests/frontend/test_swagger.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import json
+
+from . import FrontTestCase
+
+
+class SwaggerBlueprintTest(FrontTestCase):
```

Comment:
It's related to the API not the front so it should inherit from `ApiTestCase` (with proved an helper to test JSON) and located in `udata/tests/api`.